app/views/layout.jade

```diff
@@ -61,14 +61,14 @@ html
         script(src='js/controllers/devVisualizator.js')
         script(src='js/controllers/data.js')
         script(src='js/controllers/analytics.js')
-        script(src='js/controllers/session.js')
+        script(src='js/controllers/activity.js')
```

Comment:
Fixed in: https://github.com/e-ucm/rage-analytics-frontend/pull/90/commits/572bff82cfdb94cdb0df0d32cef2344ce44ca856